test/acceptance/README.md

```diff
@@ -1,25 +1,57 @@
 # Acceptance tests
 
-The idea of this tests is to automatically test the entire stack, including the library, the renderer and the integration with the backend.
+a.k.a E2E tests
 
-This is done through screenshot testing, comparing `test-cases` screenshots against reference images stored in the `references` folder.
+The idea of this tests is to automatically test the **entire stack**, including the library, the renderer and the integration with the backend.
 
-## Running the tests
+This is done through iterative screenshot testing, comparing `test` screenshots against its reference images. These tests are defined in the `test.js` file:
 
 ```
-yarn test:acceptance
+e2e
+├── test1
+│   └── test.js
+└─── test2
+│   └── test.js
+...
 ```
 
 ## Generating the references
 
 ```
-yarn test:acceptance:prepare
+yarn build
+yarn test:e2e:prepare
+```
+
+After this process the ignored file `test.hml` is created. If the test does not have a `test.png` image, it will be automatically created.
+
+```
+test1
+├── test.html
+├── test.js
```

Comment:
IMO those files should not be named `test.js`.

 I think `scenario.js` is a better name